parse/search.js

```diff
@@ -3,17 +3,54 @@ const path = require('path');
 const fs = require('fs');
 const avoParse = require('./avoParse')
 
-avoParse.getVersions().forEach(function(version) {
```

Comment:
Might be worth using these library functions

Reply:
Ah yeah my bad, will get it changed back